### ISSUE TITLE
Mycetoma Monthly Meeting 25th October 2022

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Series-1-Fenarimols#82]
_This page is draft until this text is removed_

**Time**: 25th October (12 UK, 1 EU, 9 Aus)
**Chair**: UCL
**Minute taker**: USyd
Previous meeting: #80

Recording:

### 1. Biological Evaluation (Erasmus)
- [ ] @MA-Jjingyi to post powerpoint decks from Sept meeting and from Aug meeting
- [ ] @dmitrij176 to summarise (in a new post, with a link to that post from the [wiki](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/wiki/Biological-Data-Not-Yet-Incorporated-into-Wiki)) latest in vitro and in vivo data from Aug and Sept in advance of the next meeting.
- [ ] @fantasy121 and @dmitrij176 to check that all their compounds in the Master List possess molecular weight info (and anything else that is useful/important)

### 2. New Chemistry (USyd and UCL)
- [ ] Update needed from Breaking Good team (@alintheopen @kym834 @KlementineJBS) on which molecules have been made, and which are planned, as well as an updated synthetic scheme.
- [ ] @fantasy121 to share the scheme of targets shown in #80 .

### 3. Other Samples Incoming/Desired
- [ ] @dmitrij176 to complete the analysis of the Epichem library to identify compounds useful for closing the SAR gap, and post to a new Issue before the next meeting.
- [ ] @fantasy121 to contact Rohan David, to obtain structures for his antimalarial compounds and whether they are intended for a particular biological target.
- [ ] @fantasy121 to check the compounds from @AndreaOtago and @csuth22 in OpenSourceMycetoma/series-5-benzimidazoles#3 and ensure these compounds had been assigned [MyOS Codes](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention).
- [ ] @bebi78 to advise whether all the samples he has recently contributed are in the Master List.
- [ ] @mattodd to pursue metal complexe samples from COADD and report at next meeting.

### 4. Paper Writing
- [ ] @fantasy121 will ask Alice/Peter to complete their review the draft (co-authoring) 
- [ ] @dmitrij176 to complete his inputs to the experimental section, using the offline editing of Word documents via Dropbox.
- [ ] @fantasy121 and @dmitrij176 to assemble the NMR spectra (images of spectra, and raw data files) for novel compounds - location needs to be decided.

### 5. Admin
Actions from last time:
In the last meeting it was decided to move to a time that is the same each month: 12 noon UK, 1 EU and 9 Sydney. It was also decided that
- [ ] @mattodd would generate a back-up Zoom link should USyd not be able to attend, and that
- [ ] @wwjvdsande would generate a Teams link as option number 3.
- [ ] @wwjvdsande to include these links in a new Outlook invitation for the next 12 months worth of meetings. The meetings are on the 4th Tuesday of each month
- [ ] @fantasy121 or @kym834 (whoever made them) to cancel the Outlook invites for the original series of meetings

- [ ] Breaking Good compounds @kym834 **In meeting**: @fantasy121 or @KlementineJBS to provide structures and SMILES for latest Breaking Good compounds, ensure that they are in the Master List (with codes), allowing @wwjvdsande to be able to get logD values.

- [ ] @wwjvdsande to chase @MMos2022 for the deck of her presentation and post below.

### 6. AOB

**Next Meeting:**
Time: November 22nd 12 UK, 1 EU, 9 Aus
Who is starting the Issue:
Who is chairing: 
Who is taking minutes:

**Comment by wwjvdsande** [comment]
As discussed during the MycetOS group meeting on 27 September 2022, this is the new invite for the monthly MycetOS group meetings. Our main meeting link is: 
1.	https://uni-sydney.zoom.us/j/99281549497.
When nobody from university of Sydney is able to attend and record the meeting, we will meet via the alternative link in the following order:
2.	https://ucl.zoom.us/j/99789829423
3.	[MycetOS meeting teams link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2I2MDcxZTgtYjZjOS00YTdmLTgwZTMtNWQ3OTc4NDY0MDNj%40thread.v2/0?context=%7b%22Tid%22%3a%22526638ba-6af3-4b0f-a532-a1a511f4ac80%22%2c%22Oid%22%3a%22a5ed84b8-5af1-42c6-9f8d-240f23a5d997%22%7d)

**Comment by wwjvdsande** [comment]
I added the compounds of Andrea Ortago in the mastersheet and the results of the screening. However there were no smiles. In https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/3#issuecomment-1158804057 the structures are shown can @dmitrij176 or @fantasy121 generate smiles and/or MycetOS codes for them?

**Comment by dmitrij176** [comment]
> I added the compounds of Andrea Ortago in the mastersheet and the results of the screening. However there were no smiles. In [OpenSourceMycetoma/series-5-benzimidazoles#3 (comment)](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/3#issuecomment-1158804057) the structures are shown can @dmitrij176 or @fantasy121 generate smiles and/or MycetOS codes for them?

Yes Wendy, I will let everyone know when they are added.

**Comment by dmitrij176** [comment]
@wwjvdsande Master list is now fully updated with information on Benzoxazoles

![New Benzoxazoles jpeg](https://user-images.githubusercontent.com/44785501/194729494-0ac1d75b-c8ea-4c0b-bc3e-1f15831f66cd.jpeg)

**Comment by dmitrij176** [comment]
Hi @kym834. There are only 5 (Series 2) molecules left without SMILES in the Master List:
MYOS_00315_00_01
MYOS_00316_00_01
MYOS_00317_00_01
MYOS_00318_00_01
MYOS_00319_00_01
MYOS_00320_00_01

Could you please update these entries so that we can ask Ben to calculate logD values.

**Comment by fantasy121** [comment]
@mattodd @wwjvdsande Re: Peter review of our draft manuscript, Peter asked that his review could be postponed because his free time slots for review were when he was sick. Could you proceed with your part of the reviewing process and then let Peter have a look at the draft after you are done? Sorry for the delay.

Also, I will work with Klem in the next day or two to sort out the update of the Breaking Good compound details. Klem is currently busy preparing for this year's Breaking Good Mycetoma project with undergrad students so she couldn't get back to us earlier but I will try and work with her this week to get this done.

**Comment by AndreaOtago** [comment]
> @wwjvdsande Master list is now fully updated with information on Benzoxazoles
> 
> ![New Benzoxazoles jpeg](https://user-images.githubusercontent.com/44785501/194729494-0ac1d75b-c8ea-4c0b-bc3e-1f15831f66cd.jpeg)

Hi, I saw you have updated the Otago compounds etc with your system codes, just a note - the compounds sent in by Otago, the 2-phenylpropanoic derived-compounds (2 compounds) are both the racemic mixture. This was on the sheet I sent in, but Ben may have not/uploaded misleading stereochem, definitely the R/S mixture from Otago. The Imperial team did make some enantiomers, but of different halogen regioisomers.

**Comment by mattodd** [comment]
Hi @alintheopen @kym834 @KlementineJBS I'm getting multiple requests from Schools to get involved in MycetOS this academic year. It'd be very handy to know what you've made in the last academic year and what is planned, so that we can effectively allocate novel structures. Might you be able to post an update (to OpenSourceMycetoma/Series-2-Aminothiazoles#9) on 1) made structures, 2) planned structures and 3) any improvements made to the synthetic protocol?

**Comment by AndreaOtago** [comment]
> 

Hi @dmitrij176 sorry I was just going off the visual chemdraws above, I can't find the link/where to get to the master list, and also I am unsure where the biological data file/info link is, it would be interesting to see the activity of the compounds.

**Comment by mattodd** [comment]
@dmitrij176 it's the two compounds with the stereocentres, 398 and 399. According to our convention "Different stereoisomers (or mixtures thereof) should have different MYOS codes" so these racemates can keep their codes, but the drawing needs to be altered and the SMILES/InChI/InChIKEY codes need to be re-done if they are different for racemates vs single enantiomers. @AndreaOtago the link is on the front page of this repository - http://tinyurl.com/MycetomaMols

**Comment by kym834** [comment]
> 

Have now updated the SMILES etc for these compounds in the master list.

**Comment by kym834** [comment]
> Hi @kym834. There are only 5 (Series 2) molecules left without SMILES in the Master List: MYOS_00315_00_01 MYOS_00316_00_01 MYOS_00317_00_01 MYOS_00318_00_01 MYOS_00319_00_01 MYOS_00320_00_01
> 
> Could you please update these entries so that we can ask Ben to calculate logD values.

Have now updated the SMILES etc for these compounds in the master list.

**Comment by kym834** [comment]
> Hi @alintheopen @kym834 @KlementineJBS I'm getting multiple requests from Schools to get involved in MycetOS this academic year. It'd be very handy to know what you've made in the last academic year and what is planned, so that we can effectively allocate novel structures. Might you be able to post an update (to [OpenSourceMycetoma/Series-2-Aminothiazoles#9](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/9)) on 1) made structures, 2) planned structures and 3) any improvements made to the synthetic protocol?

Hey @mattodd,

1)

I've updated the repository for series 2 with
* the compounds that were successfully made and have been shipped by the schools in May/June. https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/9
* the compounds that SSP have been working on this year. They are currently completing the characterisation and we'll know in the next week or so if they have successfully made them. https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/10

I have also updated the [wiki](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/wiki) with some extra pages that now show compounds which have been evaluated, those awaiting evaluation and those that are currently being synthesised. Hopefully this will help avoid overlap but we either need to get people to update these pages themselves or  get people posting issues in the repository with what they are planning so we can keep it updated. 

2) In terms of planned structures, the nitro compounds USyd has hopefully made this year https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/10 will be used to expand the structure through reduction of the nitro and subsequent substitution in 2023. As we are waiting on characterisation data though this isn't set in stone. I've also indicated that the 2,3 and 4-chloro compounds made last year (MYOS_00180_00_01, MYOS_00181_00_01 and MYOS_00182_00_01) and those hopefully made this year (not confirmed so no MYOS numbers yet) could be used in coupling reactions to further expand the structure. I'm working on added a story to the home page of the wiki that expands the logic behind the compounds made so far and these possible future directions.

I'm also very happy plan an administrative role and engage in discussions with people about what we make and help direct the series as well as keep the GitHub repo up to date.

3) No improvements to the synthetic protocol. I've put the detailed protocol (example from SSP 2021) into the wiki. So hopefully now visitors will find it easier.

**Comment by dmitrij176** [comment]
Updated Benzoxazoles
![New Benzoxazoles (updated)](https://user-images.githubusercontent.com/44785501/195880539-e58750e4-2468-45fb-8620-c31c30e528da.jpeg)

@mattodd @wwjvdsande the Master List is now fully updated. @bendndi would you please generate logD values for us.

**Comment by mattodd** [comment]
OK, @dmitrij176 out of interest did the chemical strings (SMILES etc) change between single enantiomer and racemate?

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib I updated the spreadsheet with the logD values etc. @bendndi was so kind to calculate the missing values

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib we are in the sydney zoom link. Is anyone from sydney joining? @fantasy121?

**Comment by bebi78** [comment]
@dmitrij176 @fantasy121 Fatty alkyl pyridinium salts (e.g., cetylpyridinium chloride/CPC) can have considerable antifungal activity (see link). 
I used N-alkyl salts of 4-picolin for the synthesis of various stilbazolium dyes as part of finished a material science project. But can be repurposed for mycetoma testing. @wwjvdsande  

https://www.mdpi.com/2309-608X/6/4/218

**Comment by MFernflower** [comment]
@bebi78 maybe a few of those stilbazolium compounds could be sent for
antifungal testing - they kind of resemble the known antifungal Miltefosine

**Comment by MFernflower** [comment]
@bebi78 @wwjvdsande


All though if I'm not mistaken miltefosine was tested and found to not
prolong larval survival

>

**Comment by MFernflower** [comment]
@dmitrij176 I was wondering if you could make one of the non commercially
available phenothiazines (series 3) if you had time to do so - something
like N-isovaleryl phenothiazine might be interesting to screen

**Comment by bebi78** [comment]
@MFernflower Yes, I will check if I still have the stilbazolium compounds and the corresponding N-alkylpicolin precursors. Maritima might also be a suitable starting compound to prepare pyridinium salts. Cetylpyridinium might be prepared analogously.

**Comment by MFernflower** [comment]
@bebi78 should just be as easy as refluxing maritima, dmf, ethyl iodide and
potash  together - Although these type of compounds tend to have toxicity
issues that limit there use to topical formulations (with the exception of
miltefosin)

**Comment by bebi78** [comment]
@MFernflower Often you don't need potash and can use ethanol as solvent, the pyridine is reactive enough to be alkylated by alkyl bromides in hot ethanol. I still have six or seven N-cetylpicolines and cetylstilbazolium dyes I can provide for testing.

**Comment by dmitrij176** [comment]
> @dmitrij176 I was wondering if you could make one of the non commercially available phenothiazines (series 3) if you had time to do so - something like N-isovaleryl phenothiazine might be interesting to screen

We will see.

**Comment by mattodd** [comment]
Hi @OpenSourceMycetoma/corecontrib - sorry to miss the meeting. Two Q's where I think we need clarification:

1) I'm having trouble figuring out from the minutes if there are new action items that we need to add to next meeting #83. Please could the minute taker check through the above minutes to see whether all actions from Oct are inluded in Nov?

2) What's the next things needed on the paper? Who is next to read the manuscript (there are still a lot of comments outstanding) and what's remaining to be done for the experimental Word file?

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande @mattodd @MA-Jjingyi Please find below a list of cetylpyridinium compounds including the new cetyl-maritima, and new niclosamide derivatives (inspired by Wendy's niclosamide paper) I can share for testing anytime.

[Niclosamide.Cetyl.docx](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/10033809/Niclosamide.Cetyl.docx)

**Comment by MA-Jjingyi** [comment]
[20221025myctOS meeting -MA.pptx](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/10056904/20221025myctOS.meeting.-MA.pptx)
the slides are here. please check.

**Comment by wwjvdsande** [comment]
Hi Ma,

Didn't we present this already last time? If so, no need to present now

Wendy

**Comment by MA-Jjingyi** [comment]
> Hi Ma, Didn't we present this already last time? If so, no need to present now Wendy From: MA-Jjingyi ***@***.***> Sent: maandag 21 november 2022 14:30 To: OpenSourceMycetoma/Series-1-Fenarimols ***@***.***> Cc: Wendy Laureijssen - van de Sande ***@***.***>; Mention ***@***.***> Subject: Re: [OpenSourceMycetoma/Series-1-Fenarimols] Mycetoma Monthly Meeting 25th October 2022 (Issue #82) Waarschuwing: Deze e-mail is afkomstig van buiten de organisatie. Klik niet op links en open geen bijlagen, tenzij u de afzender herkent en weet dat de inhoud veilig is. Caution: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe. 20221025myctOS meeting -MA.pptx<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOpenSourceMycetoma%2FSeries-1-Fenarimols%2Ffiles%2F10056904%2F20221025myctOS.meeting.-MA.pptx&data=05%7C01%7Cw.vandesande%40erasmusmc.nl%7Ca5841b8457a84841d2ab08dacbc48db7%7C526638ba6af34b0fa532a1a511f4ac80%7C0%7C0%7C638046344199692611%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=TrSoEBSvs%2BVjg6lHytgW8qO6AL%2BXQoeaTehb1mHBwvQ%3D&reserved=0> the slides are here. please check. - Reply to this email directly, view it on GitHub<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOpenSourceMycetoma%2FSeries-1-Fenarimols%2Fissues%2F82%23issuecomment-1322068723&data=05%7C01%7Cw.vandesande%40erasmusmc.nl%7Ca5841b8457a84841d2ab08dacbc48db7%7C526638ba6af34b0fa532a1a511f4ac80%7C0%7C0%7C638046344199692611%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=j1G2dFVdgt0s3qnYzfEhTWZPvV30yeFSiE9r3D%2FKyz4%3D&reserved=0>, or unsubscribe<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAIVWIOTQ4RVK2AJL6SCC5ADWJN2PFANCNFSM6AAAAAAQW6AR3Y&data=05%7C01%7Cw.vandesande%40erasmusmc.nl%7Ca5841b8457a84841d2ab08dacbc48db7%7C526638ba6af34b0fa532a1a511f4ac80%7C0%7C0%7C638046344199692611%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=0IFDZ61Ge5caWer3%2FUT8X6IUmFDJzaTONLQNtJya00g%3D&reserved=0>. You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

@wwjvdsande we did present it last month. and I forgot to upload them last month. so, i upload them now.

**Comment by AndreaOtago** [comment]
> 

Finally getting around to having a look at the biological results from the Otago benzoxazoles out of interest, and I see a few are quite good, and also looks like the best here were some of the worst/inactive in the OSN/DNDi/parasite screen, so interesting!

**Comment by dmitrij176** [comment]
![Potency results (July-Sept 2022) Orange Box jpeg](https://user-images.githubusercontent.com/44785501/203572032-80519b39-f400-4cc0-b379-dd098cdc302a.jpeg)
[Potency results (July-Sept 2022) Orange Box.zip](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/10076418/Potency.results.July-Sept.2022.Orange.Box.zip)
